### PR TITLE
TRIVIAL: Allow override source branch for Dev release workflow

### PR DESCRIPTION
**Pull request by jaceksan**
Plus gooddata-dbt - generate short table ID

Plus small fixes

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: .github/workflows/dev-release.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@ name: Dev release from master
 
 on:
   workflow_dispatch:
+    inputs:
+      BRANCH_NAME:
```
Comment (hkad98) [inline .github/workflows/dev-release.yaml:6]: Please, add master as default.

Comment (jaceksan) [reply .github/workflows/dev-release.yaml:6]: added

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR533

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/307/ : FAILURE in 1m 36s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/457/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/461/ : SUCCESS in 9s

**Pull request merged**